Python/Product/DebuggerHelper/trace.cpp

```diff
@@ -594,4 +594,17 @@ void *EvalFrameFunc(void* f, int throwFlag)
     return nullptr;
 }
 
+typedef void* (*_PyFrameEvalFunction_39)(void*,void*, int);
```

Comment:
I should put a comment here saying why this is different.

Reply:
would help, yes